README.md

```
@@ -337,7 +337,65 @@ TERMNET_ENFORCE_SLO=true python -m pytest tests/test_slo_guard_pytest.py
 
+## 🤖 How to Run Autonomous Project Mode (DMVL)
+
+TermNet's **Dual-Model Validator Loop (DMVL)** enables autonomous project creation where:
+- **Claude** proposes edits (via `say`/`planner`)
+- **Qwen** verifies claims (lint/tests/commands via Computer-Use)
+- **If all OK** → PR opened + labels
+- **If fail** → receipts + advice
+
+### Environment Setup
+
+```bash
```

markdownlint-cli2 (0.18.1)

Code fence style
Expected: tilde; Actual: backtick

(MD048, code-fence-style)